packages/react-components/react-select/src/stories/SelectAppearance.stories.tsx

```diff
@@ -1,52 +1,90 @@
 import * as React from 'react';
 import { Select } from '../index';
 import { useId } from '@fluentui/react-utilities';
+import { makeStyles, mergeClasses, shorthands } from '@griffel/react';
+import { tokens } from '@fluentui/react-theme';
+
+const useStyles = makeStyles({
+  base: {
+    display: 'flex',
+    flexDirection: 'column',
+    maxWidth: '500px',
+  },
+
+  field: {
+    display: 'grid',
+    gridRowGap: tokens.spacingVerticalXXS,
+    marginTop: tokens.spacingVerticalMNudge,
+    ...shorthands.padding(tokens.spacingVerticalMNudge, tokens.spacingHorizontalMNudge),
+  },
+
+  filledLighter: {
+    backgroundColor: '#8a8a8a',
+    '> label': {
+      color: '#ffffff',
+    },
+  },
+  filledDarker: {
+    backgroundColor: '#8a8a8a',
+    '> label': {
+      color: '#ffffff',
```

Comment:
quick question -- when I did this earlier for combobox, I noticed that white text actually has much less contrast (3.45:1) vs. black text (6.08:1). The contrast calc isn't perfect, but I wonder if we should leave the text black, and maybe bold it instead.